src/stories/Library/search-result-page/search-result-page-skeleton.scss

```diff
@@ -0,0 +1,15 @@
+.search-result-page__skeleton-facet-line {
```

Comment:
Please consider what approach to creating new classes in skeleton screens is/should be.

Here we use `search-result-page__skeleton-facet-line` which looks BEM-like. Above you use `skeleton-disclosure` which does not appear BEM-like. It would nice with some consistency for others to follow.

I have not formed any definitive opinion myself on the topic. The BEM-like structure makes it clear to see where the element belongs and should avoid clashes. Then again having a BEM structure spread across multiple files seems to go against the method.

Ideally I would like to avoid any should custom classes and thus these problems but perhaps that is not possible. 

I wonder if we could avoid the problem entirely by not having any custom behavior:

1. Disclosures are so far down the page they might not even affect CLS - even across devices.
2. I could not find any mobile/desktop switching for the facet line in our production code but we have it for the skeletons. Could we instead make each skeleton facet element shorter instead of full width?